draft-ietf-suit-firmware-encryption.md

```diff
@@ -610,39 +587,12 @@ by the sender:
 
 ### CDDL
 
-The CDDL for the COSE_Encrypt_Tagged structure is shown in {{cddl-esdh}}.
+The CDDL for the ECDH-ES+AES-KW binary is shown in {{cddl-esdh}}.

```

Comment:
The CDDL below is only valid for binaries using ECDH-ES+AES-KW for key exchange algorithm.